config/crd/bases/app.example.com_defaultremediations.yaml

```diff
@@ -0,0 +1,57 @@
+---
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  annotations:
+    controller-gen.kubebuilder.io/version: v0.4.1
+  creationTimestamp: null
+  name: defaultremediations.app.example.com
```

Comment:
replace `app.example.com`

Reply:
actually I now see we have each CR twice?

Reply:
Indeed - I've missed deleting after changing group name